source/guides/zepto-api.md

```diff
@@ -317,7 +317,7 @@ Common use cases:
 
 Provides the ability to send a Payment Request (get paid) to any Contact that has an accepted Agreement in place.
 
-To send a Payment Request to a Contact using the API, you must first have an accepted [Agreement](doc:zepto-api#Zepto-API-Agreements) with them.
+To send a Payment Request to a Contact using the API, you must first have an accepted [Agreement](https://help.zepto.money/en/articles/3094575-au-agreements) with them.
```

Comment:
Not sure which to use but IIRC we're phasing out use of `zepto.money`. 

I wonder if it was originally pointing here:
https://docs.zeptopayments.com/reference/listoutgoingagreements

Reply:
That works also, updated.